cclib/parser/gaussianparser.py

```diff
@@ -1554,6 +1554,74 @@ def extract(self, inputfile, line):
                 self.nocoeffs = nocoeffs
                 self.nooccnos = nooccnos
 
+
```

Comment:
Can you add a comment above here showing what NSO block looks like? There's an example for `Natural Orbital Coefficients` above.

Reply:
Looks like this was added.